returnn/native_op.py

```diff
@@ -340,7 +340,6 @@ def map_layer_inputs_to_op(cls, Z, V_h, i):
     Map.
     """
     import returnn.util.basic
-    assert returnn.util.basic.BackendEngine.is_theano_selected()
```

Comment:
X

Reply:
For me the comment was shown

Reply:
Only here but not in the file diff view for me. But anyway.